records_mover/records/prep_and_load.py

```diff
@@ -25,9 +25,14 @@ def prep_and_load(tbl: TargetTableDetails,
         #
         #  Cannot COPY into nonexistent table
         driver = tbl.db_driver(db)
+        loader = driver.loader()
+        # TODO: Can this method maybe take a loader rather than a
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker